packages/utils/src/validation.ts

```diff
@@ -1,15 +1,18 @@
-import * as v from 'valibot'
 import { getGlobalConfig } from './config'
+import z from 'zod'
 
-export const vHttpsUrl = v.pipe(
-  v.string(),
-  v.url(),
-  v.check((url) => {
-    const { allowInsecureUrls } = getGlobalConfig()
-    return allowInsecureUrls ? url.startsWith('http://') || url.startsWith('https://') : url.startsWith('https://')
-  }, 'url must be an https:// url')
-)
-export const vInteger = v.pipe(v.number(), v.integer())
+export const vHttpsUrl = z
```

Comment:
We should probably rename these prefixes to zHttpsUrl, etc..

Reply:
Yeah, I had doubts about that, but then I thought the `v` prefix could work to mean `validation`

But yup, agree! we can also use `z` everywhere instead

Reply:
Used the `z` prefix for all zod schema variables  
also renamed files with `v-*` prefix to `z-*`